.stylelintrc.json

```diff
@@ -34,6 +34,7 @@
     "color-function-notation": "legacy",
     "value-keyword-case": ["lower", {
       "ignoreProperties": ["/font-family/"]
-    }]
+    }],
+    "custom-property-empty-line-before": null
```

Comment:
[inform]: Disabled this rule because I think adding indents between abstract variables improves readability.